src/compiler/extensions/postprocess.js

```diff
@@ -1,5 +1,241 @@
 import util from '../utils/builder';
 
+function toStringLiteral(val) {
+  return `'${val.replace('\'', '\\\'')}'`;
+}
+
+function writeVals(indexes, entities, out) {
```

Comment:
Some comments about what this function does (and what params are expected) would be helpful here.